src/tabs/worldMap/WorldMapTabController.tsx

```diff
@@ -0,0 +1,15 @@
+import {TabController} from "../TabController";
+import {WorldMapTabState} from "./WorldMapTabState";
+
+export class WorldMapTabController extends TabController<WorldMapTabState> {
+
+  // TODO: figure out how to send the data
```

Comment:
Maybe the input could simply be 3 text boxes (or a more appropriate control for number input, but I don't think there is one in ReactXP), for every coordinate - or just 2 for the "top-down view" version. Or both of the options, where the Y input can be toggled with a "slice world with an horizontal plane at the specified Y" (or similar description) checkbox. When the option is not checked - thus the user wants the top-down view, no cutting - maybe we could send `-1` for Y, then the server world check if Y negative and, if yes, compute the proper Y of each block (and optionally adjust the color, as @mkienenb suggested on Slack).